### PR TITLE
Add random table suffix in TestHiveTransactionalTable

### LINKED ISSUE #4089: Broken io.prestosql.tests.hive.TestHiveTransactionalTable.testReadFullAcid
```
2020-06-19T06:36:14.6669489Z tests               | 2020-06-19 12:21:14 INFO: FAILURE     /    io.prestosql.tests.hive.TestHiveTransactionalTable.testReadFullAcid (Groups: storage_formats, hive_transactional) took 50.4 seconds
2020-06-19T06:36:14.6738598Z tests               | 2020-06-19 12:21:14 SEVERE: Failure cause:
2020-06-19T06:36:14.6738853Z tests               | java.lang.AssertionError: tableCompactionsIds are empty expected [false] but found [true]
2020-06-19T06:36:14.6776353Z tests               | 	at org.testng.Assert.fail(Assert.java:94)
2020-06-19T06:36:14.6776903Z tests               | 	at org.testng.Assert.failNotEquals(Assert.java:513)
2020-06-19T06:36:14.6777233Z tests               | 	at org.testng.Assert.assertFalse(Assert.java:63)
2020-06-19T06:36:14.6777581Z tests               | 	at io.prestosql.tests.hive.TestHiveTransactionalTable.compactTableAndWait(TestHiveTransactionalTable.java:203)
2020-06-19T06:36:14.6777931Z tests               | 	at io.prestosql.tests.hive.TestHiveTransactionalTable.testReadFullAcid(TestHiveTransactionalTable.java:95)
2020-06-19T06:36:14.6778304Z tests               | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2020-06-19T06:36:14.6778641Z tests               | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2020-06-19T06:36:14.6778981Z tests               | 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2020-06-19T06:36:14.6779311Z tests               | 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2020-06-19T06:36:14.6779643Z tests               | 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
2020-06-19T06:36:14.6779973Z tests               | 	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
2020-06-19T06:36:14.6780297Z tests               | 	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
2020-06-19T06:36:14.6780879Z tests               | 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
2020-06-19T06:36:14.6781211Z tests               | 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2020-06-19T06:36:14.6781543Z tests               | 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2020-06-19T06:36:14.6781877Z tests               | 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2020-06-19T06:36:14.6782217Z tests               | 	at java.base/java.lang.Thread.run(Thread.java:834)
```

### PR DESCRIPTION
**Pull request by losipiuk**
Attempt to fix #4089

### CODE REVIEW

File: presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java [hunk 1]
```diff
@@ -57,7 +58,7 @@ public void testReadFullAcid(boolean isPartitioned, BucketingType bucketingType)
             throw new SkipException("Presto Hive transactional tables are supported with Hive version 3 or above");
         }
 
-        String tableName = "test_full_acid_table_read";
+        String tableName = format("test_full_acid_table_read_%s", randomTableSuffix());
```
Comment (sopel39) [inline presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java:61]: just use `"test_full_acid_table_read_" + randomTableSuffix()`
(see other tests)

File: presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java [hunk 2]
```diff
@@ -107,7 +108,7 @@ public void testReadInsertOnly(boolean isPartitioned, BucketingType bucketingTyp
             throw new SkipException("Presto Hive transactional tables are supported with Hive version 3 or above");
         }
 
-        String tableName = "test_insert_only_table_read";
+        String tableName = format("test_insert_only_table_read_%s", randomTableSuffix());
```
Comment (findepi) [inline presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java:111]: per code style, we do not use format when there is only trailing %s, concat instead

**Comment by wendigo** [comment]
@findepi I've been testing whether if I drop table in Hive (HDP2.6, 3.1) compactions history will be removed as well and it was. @losipiuk was able to reproduce though but I don't know the details.

**Comment by losipiuk** [comment]
> @findepi I've been testing whether if I drop table in Hive (HDP2.6, 3.1) compactions history will be removed as well and it was. @losipiuk was able to reproduce though but I don't know the details.

Just once. Now I run it from time to time with debugger hooked in - with no success.

**Comment by losipiuk** [comment]
DOes not help with #4089 and is there is no strict need for this change. Closing

**Pull request closed without merging**